app/config/wallabag.yml

```diff
@@ -125,15 +125,15 @@ wallabag_core:
             value: 0
             section: import
         -
-            name: piwik_enabled
+            name: matomo_enabled
             value: 0
             section: analytics
         -
-            name: piwik_host
-            value: v2.wallabag.org
+            name: matomo_host
+            value: matomo.wallabag.org
```

Comment:
I changed this line because in fact, we need to configure the URL of the matomo instance there.